Chapter8t11/Operator.cpp

```diff
@@ -0,0 +1,65 @@
+#include "Operator.h"
+#include "CFourClass.h"
+
+void Operator::CFour() {
+    ConnectFour b;
+    b.resetBoard();
+    b.drawBoard();
+    bool playing = true;
+    bool blueTurn = true;
+    int moves = 0;
+
+    while (b.checkWin() == white && moves < 42) {
+        int choice;
+        if (blueTurn) {
+            std::cout << std::endl;
+            std::cout << "It is Blue's Turn! Please choose a column (1-7): ";
```

Comment:
Any string used should be stored in an enum. This makes it easier to reuse strings later, and if you use them in multiple places, you can change one line to change many resulting strings. Can you add the enum into the ConnectFour class.